From 5fe83bd36c035099a6d721a8e42cf3d04a1ce2b6 Mon Sep 17 00:00:00 2001
From: Christian Marangi <ansuelsmth@gmail.com>
Date: Sun, 12 May 2024 14:36:32 +0200
Subject: [PATCH] Use %zu and %zd for ssize_t and size_t

Use %zu and %zu for ssize_t and size_t to fix compilation warning on
64Bit.

Signed-off-by: Christian Marangi <ansuelsmth@gmail.com>
---
 debugfs.c              | 2 +-
 hif/pcie/8964/tx_ndp.c | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

--- a/debugfs.c
+++ b/debugfs.c
@@ -1394,7 +1394,7 @@ done:
 				 priv->reg_value);
 	else
 		len += scnprintf(p + len, size - len,
-				 "error: %d(%u 0x%08x 0x%08x)\n",
+				 "error: %zd(%u 0x%08x 0x%08x)\n",
 				 ret, priv->reg_type, priv->reg_offset,
 				 priv->reg_value);
 
--- a/hif/pcie/8964/tx_ndp.c
+++ b/hif/pcie/8964/tx_ndp.c
@@ -336,7 +336,7 @@ int pcie_tx_init_ndp(struct ieee80211_hw
 
 	if (sizeof(struct pcie_tx_ctrl_ndp) >
 	    sizeof(tx_info->driver_data)) {
-		wiphy_err(hw->wiphy, "driver data is not enough: %d (%d)\n",
+		wiphy_err(hw->wiphy, "driver data is not enough: %zu (%zu)\n",
 			  sizeof(struct pcie_tx_ctrl_ndp),
 			  sizeof(tx_info->driver_data));
 		return -ENOMEM;
